index.html

```diff
@@ -196,5 +196,29 @@ <h1>Hey There. <br>
             </div>
         </section>
     </main>
+    <section id="contact" class="footer">
+        <form class="contact-form" action="https://formspree.io/f/xlevqbal" method="post">       
+            <h2 class="touch">
+              I'm always<br>interested in hearing about new projects, so<br>if you'd like to chat<br>please get in touch.
+            </h2>
+            <div class="inputs-contact-form">
+              <input id="contact-form-name" class="contact-form-text" name="Fullname" type="text" maxlength="30" placeholder="Full name" required>
+              <input id="contact-form-email" class="contact-form-text" name="Email" type="email" placeholder="Email" required>
+              <textarea name="Comment" id="contact-form-message" cols="15" rows="10" maxlength="500" placeholder="Place Text Here" required></textarea>
+              <div class="submit-container">
+                <input id="contact-form-button" class="link-styler" type="submit" name="Get in touch" value="Get in touch">
+              </div>
+            </div>
+        </form>
+        <div class="line"><img src="./icons/Rectangle 42.png" alt=""></div>
+    <ul class="logo1">
+        <li><a href="https://github.com/UsamaMasood12"><img src="./icons/C.png" alt=""></a></li>
+        <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./icons/I.png" alt=""></a></li>
+        <li><a href="#"><img src="./icons/V.png" alt=""></a></li>
+        <li><a href="https://twitter.com/Usama____Masood"><img src="./icons/T.png" alt=""></a></li>
+        <li><a href="#"><img src="./icons/M.png" alt=""></a></li>
+    </ul>
+    <div class="line2"><img src="./icons/Shape.png" alt=""></div>
+    </section>
```

Comment:
Great job implementing the changes required by a previous reviewer! 🚀 

- [ ] **[OPTIONAL]:** I suggest you fix your indentation for the entire file in order to keep it cleaner and improve its readability. 🤓 🛠️